tests/settings.py

```diff
@@ -10,8 +10,6 @@
 
 SECRET_KEY = 'secret'
 
-RESOLWE_API_HOST = 'localhost'
```

Comment:
I think it doesn't really scale to have the test project have all possible setting values set and serve as a reference. I think we would need proper configuration documentation, hence I opened #189.